### ISSUE TITLE
Prepare a better guide for running mev-boost with a builder in a local private devnet

### ISSUE THREAD

**Issue by Ruteri** [flashbots/mev-boost#180]
Current guide https://github.com/flashbots/mev-boost/blob/main/docs/geth-pos-privnet.md is outdated and too hard to follow.
We need a clear, possibly automated way to prepare a local devnet.

**Comment by Ruteri** [comment]
Good starting point would be https://github.com/ajsutton/merge-scripts/tree/main/localnet

**Comment by Ruteri** [comment]
Also relevant: https://github.com/flashbots/mev-boost/pull/187

**Comment by metachris** [comment]
https://github.com/flashbots/mev-boost/blob/main/docs/geth-pos-privnet.md was recently updated. is this task done?

**Comment by Ruteri** [comment]
I don't think it's quite there yet - it's still convoluted and hard to follow.
